RELEASE-NOTES.md

```diff
@@ -17,6 +17,7 @@
 * Optimize snowpark dependency search to lower the size of .zip artifacts and
   the number of anaconda dependencies for snowpark projects.
 * Added support for fully qualified stage names in stage and git execute commands.
+* Fixed `snow app run` not upgrading the application after `snow app deploy`.
```

Comment:
Suggest that this has a bit more information so that the docs team can use it to prepare docs for the release.